system-test/pubsub.ts

```diff
@@ -185,7 +187,7 @@ describe('pubsub', () => {
 
     it('should be created and deleted', done => {
       const TOPIC_NAME = generateTopicName();
-      pubsub.createTopic(TOPIC_NAME, err => {
+      pubsub.createTopic(TOPIC_NAME, (err: ServiceError | null) => {
```

Comment:
Same feedback here.  You shouldn't need to use explicit types for the parameters in the callback.  Something doesn't smell right.